src/MakeJsonCommand.php

```diff
@@ -56,13 +60,24 @@ class MakeJsonCommand extends WP_CLI_Command {
 	 *     # Create JSON files for my-plugin-de_DE.po and leave the PO file untouched.
 	 *     $ wp i18n make-json my-plugin-de_DE.po /tmp --no-purge
 	 *
+	 *     # Create JSON files with mapping
+	 *     $ wp i18n make-json languages --use-map=build/map.json
+	 *
+	 *     # Create JSON files with multiple mappings
+	 *     $ wp i18n make-json languages '--use-map=["build/map.json","build/map2.json"]'
```

Comment:
Interesting, hadn't thought about that. To pass multiple files, I would have imagined `--use-map=build/map.json,build/map2.json` to be more intuitive. Is that also supported?

Reply:
No, because that would cause problems with passing objects.

Previously, when only paths were accepted, that was the way.

But not, it would not be possible to distinguish between the comma inside an object and the comma between files, at least not trivially. Hence my suggestion of a regex mentioned here https://github.com/wp-cli/i18n-command/pull/284#discussion_r714748833